proto/osmosis/incentives/genesis.proto

```diff
@@ -18,4 +18,13 @@ message GenesisState {
     (gogoproto.stdduration) = true,
     (gogoproto.moretags) = "yaml:\"lockable_durations\""
   ];
+  repeated GenesisReward genesis_reward = 4 [ (gogoproto.nullable) = false ];
+  repeated PeriodLockReward period_lock_reward = 5
+      [ (gogoproto.nullable) = false ];
 }
+
+message GenesisReward {
```

Comment:
Oh interesting, this exports to genesis undistributed rewards. The cosmos SDK's staking auto distributes the rewards at export genesis.

What do you think makes the overall code simpler?

Reply:
Do you mean adding distribution logic at export genesis?

Reply:
Yeah. (I'm also fine not doing it, just a question of overall simplicity)